kolibri/core/assets/src/views/ExamReport/index.vue

```diff
@@ -176,6 +186,7 @@
     },
     methods: {
       handleNavigateToQuestion(questionNumber) {
+        console.log(questionNumber);
```

Comment:
`console.log` snuck in. It's odd that the linter didn't complain.

Reply:
At some point in the relatively near past (~1 year) our linter seemed to stop caring about console.log - I don't know why yet, but I don't like it.